spec/features/admin/subscriptions_spec.rb

```diff
@@ -224,17 +224,42 @@ def open_subscription_products_panel
         select2_select "Australia", from: "bill_address_country_id"
         select2_select "Victoria", from: "bill_address_state_id"
 
+        # Making sure that once set with empty it actually changes, and add the phone back
+        fill_in "bill_address_phone", with: ''
+        expect(page).to have_input "bill_address_phone", with: ''
+        expect(page).to have_content 'can\'t be blank', count: 1
+        fill_in "bill_address_phone", with: '0400 123 456'
+
         # Use copy button to fill in ship address
         click_link "Copy"
         expect(page).to have_input "ship_address_firstname", with: 'Freda'
         expect(page).to have_input "ship_address_lastname", with: 'Figapple'
         expect(page).to have_input "ship_address_address1", with: '7 Tempany Lane'
+        expect(page).to have_input "ship_address_city", with: 'Natte Yallock'
+        expect(page).to have_input "ship_address_zipcode", with: '3465'
+        expect(page).to have_input "ship_address_phone", with: '0400 123 456'
+        expect(page).to have_input "ship_address_country_id", with: 'Australia'
+        expect(page).to have_input "ship_address_state_id", with: 'Victoria'
 
         click_button('Next')
         expect(page).to have_content 'NAME OR SKU'
         click_button('Next')
         expect(page).to have_content 'Please add at least one product'
 
+        # Adding a product when the quantity is zero
+        add_variant_to_subscription test_variant, 0
+        within 'table#subscription-line-items tr.item', match: :first do
```

Comment:
The build is failing due to these lines, in which a variant with zero quantity is being added. If one adds `byebug` and removes the `headless` from `spec_helper.rb` (see our [wiki for some tips on debugging](https://github.com/openfoodfoundation/openfoodnetwork/wiki/Testing-and-Rspec-Tips#debugging--tests)) we can check what's happening on the browser:

![image](https://user-images.githubusercontent.com/49817236/117130358-da5b1000-ad97-11eb-96d8-8259d40bafcf.png)

Above we can see that the line item is not being added to the order, because the UI is not accepting zero as a value; opening the console we can check that this input field has a condition of `min=1`, see:
![Peek 2021-05-05 11-50](https://user-images.githubusercontent.com/49817236/117130615-3887f300-ad98-11eb-94ef-d9c2ca8d583d.gif)

So, I think this test case is useful but it is not valid right now, because it expects a line item to be added. We could consider removing the `within` block - it's causing this failure on the build:

`        expected to find css "#subscription-line-items .item" 1 time but there were no matches`

and replace it with an assertion expecting no line item to be added, when quantity is set to zero.